Client/src/Components/TabPanels/Account/TeamPanel.jsx

```
@@ -12,389 +12,298 @@ import Select from "../../Inputs/Select";
-			if (validation.error) {
-				updatedErrors.email = validation.error.details[0].message;
-			} else {
-				delete updatedErrors.email;
-			}
-			return updatedErrors;
-		});
-	};
+      if (validation.error) {
+        updatedErrors.email = validation.error.details[0].message;
+      } else {
+        delete updatedErrors.email;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)